<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverlayDialogBox</class>
 <widget class="QWidget" name="OverlayDialogBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diner Tablet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton  {
    min-height: 2.0em;
    margin: 0 2px 0 1px;
    color: white;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #581702, stop: 1 #7f3016); 
    border-style: outset;
    border-radius: 10px;
    border-width: 1px;
    border-color: #3e170b;
}
 
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #aa401e stop: 1 #581702);
}

QPushButton:!enabled {
    color: gray;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #581702, stop: 1 #7f3016); 
    border-style: outset;
    border-radius: 10px;
    border-width: 1px;
    border-color: #3e170b;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="labelMsg">
     <property name="text">
      <string>Text Message</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="styleSheet">
      <string notr="true">#line {
    border: 1px solid black;
    border-radius: 10px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #581702, stop: 1 #7f3016); 
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    font: bold 16px;
}</string>
       </property>
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="styleSheet">
        <string notr="true">
QPushButton {
    font: bold 16px;
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
